importers/deb/deb_test.go

```diff
@@ -0,0 +1,434 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deb
+
+import (
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func sha256sum(path string) ([32]byte, error) {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return [32]byte{}, err
+	}
+
+	return sha256.Sum256(data), nil
+}
+
+func testImages() ([]*Archive, error) {
+	gLinuxRepo := NewRepo("testdata")
```

Comment:
I think this should be debRepo

Reply:
Done. Note that these tests were copy-pasted from the targz importer which still contains x20/glinux references.